scripts/tests/pages-artifact.test.py

```
@@ -1998,6 +1998,149 @@ def test_river_actions_expire_pending_hash_navigation(self) -> None:
+            probe = f"""
+const result = {{ value: null }};
+globalThis.location = {{
+  pathname: {json.dumps(pathname)},
+  search: {json.dumps(search)},
+  hash: {json.dumps(fragment)},
+  replace: (value) => {{ result.value = value; }},
+}};
+{script}
+console.log(JSON.stringify(result.value));
+"""
+            completed = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)